### ISSUE TITLE
Error: WMI.GetObject.Get not implemented!

### ISSUE THREAD

**Issue by Idriel** [CapacitorSet/box-js#55]
```
Using a 10 seconds timeout, pass --timeout to specify another timeout in seconds
/usr/local/lib/node_modules/box-js/lib.js:28
		throw new Error(message);
		^

Error: WMI.GetObject.Get not implemented!
    at Object.kill (/usr/local/lib/node_modules/box-js/lib.js:28:9)
    at Object.get (/usr/local/lib/node_modules/box-js/emulator/WMI.js:82:8)
    at eval (eval at <anonymous> (sample.js:272:1), <anonymous>:1:35)
    at sample.js:272:1
    at ContextifyScript.Script.runInContext (vm.js:59:29)
    at ContextifyScript.Script.runInNewContext (vm.js:65:15)
    at Object.runInNewContext (vm.js:135:38)
    at Object.<anonymous> (/usr/local/lib/node_modules/box-js/analyze.js:428:5)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)

 * If the error is about a weird "Unknown ActiveXObject", try --no-kill.
 * Otherwise, report a bug at https://github.com/CapacitorSet/box-js/issues/ .

```

**Comment by CapacitorSet** [comment]
Can you share the sample? If needed, you can find my e-mail in my profile.

**Comment by Idriel** [comment]
Sure. I'll send it to your email.

**Comment by CapacitorSet** [comment]
Thank you, I fixed this in v1.9.14.
